### ISSUE TITLE
Octopi parsing IPv6 address incorrectly

### ISSUE THREAD

**Issue by k8tek** [guysoft/OctoPi#797]
In the shell, octopi is reporting the IPv6 address to access the UI incorrectly. 

IPv6 addresses must be contained in brackets[] to open them directly in the browser. 

In the attached picture, you can see the URL displayed to the access does not contain required brackets, leading to confusion when novices try to access the webpage, especially when they don’t understand the url presented looks different than what they’re expecting. 

![6D7175B1-C84E-4883-81F1-669ECE9DD1B7](https://user-images.githubusercontent.com/69276965/200176506-e4e689d7-a7df-4bfb-9e66-776da98f77b5.jpeg)

**Comment by guysoft** [comment]
Yep, i actually use IPv6 but usually the hostname works, never had to copy one.
It will need to be changed here: https://github.com/guysoft/OctoPi/blob/devel/src/modules/octopi/start_chroot_script#L230
Can't find a decent bash validation script for that, PRs welcome
